app/src/main/java/com/fourlastor/dante/MainActivity.java

```diff
@@ -49,14 +67,26 @@ protected void onStart() {
                 "\n" +
                 "<blockquote><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus magna. Cras in mi at felis aliquet congue. Ut a est eget ligula molestie gravida. Curabitur massa. Donec eleifend, libero at sagittis mollis, tellus est malesuada tellus, at luctus turpis elit sit amet quam. Vivamus pretium ornare est.</p></blockquote>\n" +
                 "\n" +
+                "<p><img src='https://github.com/fourlastor/dante/raw/master/images/screencast.gif' /></p>" +
+                "\n" +
                 "<h3>Header Level 3</h3>\n" +
                 "\n" +
                 "<ul>\n" +
                 "   <li>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</li>\n" +
                 "   <li>Aliquam tincidunt mauris eu risus.</li>\n" +
                 "</ul>";
 
-        Spanned htmlText = flavoredHtml.parse(rawHtml);
-        html.setText(htmlText);
+        new AsyncTask<Void, Void, Spanned>() {
+
+            @Override
+            protected Spanned doInBackground(Void... params) {
+                return flavoredHtml.parse(rawHtml);
+            }
+
+            @Override
+            protected void onPostExecute(Spanned spanned) {
+                html.setText(spanned);
```

Comment:
It depends. If the AsyncTask takes a long time and the activity goes in the background, this will effectively leak the parent activity. It will/should eventually be GC'd but then again, you're still leaking.